canonical-web-and-design/buy.ubuntu.com
# Setup local dev

## Done

- Added current buy.ubuntu.com theme to the repo
- Set up the project for local development
- Updated readme with instructions for local development

## QA

- Clone this repo, and checkout this branch
- Follow the instructions in the readme
- See that a browser tab opens automatically and presents a version of the Shopify theme running on [localhost:3000](http://localhost:3000/)

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/5781
